app/views/search/_result.html.erb

```diff
@@ -63,7 +63,7 @@
       <ul class="list-local-locations">
       <% result.location.each do |location| %>
         <li class="result-local-location">
-          <%= location[0] %>: <%= location[1] %> (not live)
```

Comment:
This line shows us the cached status from EDS. It is displayed if anything goes wrong when requesting the realtime information. The text should reflect that it is not realtime information. I don't particularly care what exactly we say, but we should make it clear it is not realtime.